front/lock.form.php

```diff
@@ -58,7 +58,31 @@
          }
       }
    }
+
+   //Execute hook to unlock fields managed by a plugin, if needed
+   Plugin::doHookFunction('unlock_fields', $_POST);
+
+} else if (isset($_POST['itemtype']) && isset($_POST["purge"])) {
+   $itemtype    = $_POST['itemtype'];
```

Comment:
I agree with that; but this is not the point of this PR. See https://github.com/glpi-project/glpi/issues/2394